kernel/Cargo.toml

```diff
@@ -8,3 +8,14 @@ edition = "2018"
 tock-registers = { path = "../libraries/tock-register-interface" }
 tock-cells = { path = "../libraries/tock-cells" }
 tock-tbf = { path = "../libraries/tock-tbf" }
+
+# In general, Tock discourages the use of cargo features. However for certain
+# kernel crate configuration, we have not found reasonable alternatives to
+# setting boolean values in kernel/src/config.rs. To make those settings configurable
+# without requiring users to modify the kernel crate itself, we allow configuration of
+# the values of the global CONFIG constant in kernel/src/config.rs via these features.
+# Other uses of cargo features are disallowed.
```

Comment:
It would be great to capture

```
[dependencies]
kernel = { path = "../../kernel", features = ["no_debug_panics"] }
```

here for those of us who do not use rust features ever.

Reply:
I added it to the comment